### PR TITLE
update API download URL for Waterfall and Velocity

### PR DESCRIPTION
**Pull request by thecmdradama**
Figured why just stop with Paper and make the API changes to Waterfall and Velocity as well. 

I have tested the rewrite of the velocity template too and it is working

### CODE REVIEW

File: minecraft-velocity/minecraft-velocity.json [hunk 1]
```diff
@@ -37,48 +53,30 @@
...
         "stop": "shutdown",
-        "environmentVars": {},
+        "command": "${java} -Xmx${memory}M -Xms${memory}M -Dterminal.jline=false -Dterminal.ansi=true -Dlog4j2.formatMsgNoLookups=true -jar velocity.jar",
+        "workingDirectory": "",
         "pre": [],
-        "post": []
+        "post": [],
+        "environmentVars": {}
     },
-    "supportedEnvironments": [{
```
Comment (LordRalex) [inline minecraft-velocity/minecraft-velocity.json:77]: Supported environments needs to stay

**Comment by thecmdradama** [comment]
Ah whoops... Have added supported Environment back in.

File: minecraft-velocity/minecraft-velocity.json [hunk 2]
```diff
@@ -37,48 +53,33 @@
...
         "pre": [],
-        "post": []
+        "post": [],
+        "environmentVars": {}
     },
     "supportedEnvironments": [{
         "type": "standard"
     }],
-    "name": "minecraft-velocity",
-    "requirements": {
```
Comment (LordRalex) [inline minecraft-velocity/minecraft-velocity.json:81]: These need to stay. They are related to the next release.

Comment (thecmdradama) [reply minecraft-velocity/minecraft-velocity.json:81]: Hmm.. not sure why the requirements were deleted as well. They have been restored too.

**Pull request merged**